Tests/cffLib/specializer_test.py

```diff
@@ -918,28 +919,21 @@ def test_maxstack(self):
         self.assertEqual(get_specialized_charstr(test_charstr), xpct_charstr)
 
 
-class CFF2VFTestSpecialize(unittest.TestCase):
-
-    def __init__(self, methodName):
-        unittest.TestCase.__init__(self, methodName)
-        # Python 3 renamed assertRaisesRegexp to assertRaisesRegex,
-        # and fires deprecation warnings if a program uses the old name.
-        if not hasattr(self, "assertRaisesRegex"):
-            self.assertRaisesRegex = self.assertRaisesRegexp
+class CFF2VFTestSpecialize(DataFilesHandler):
 
     @staticmethod
     def get_test_input(test_file_or_folder):
         path, _ = os.path.split(__file__)
         return os.path.join(path, "data", test_file_or_folder)
 
     def test_blend_round_trip(self):
-        otfvf_path = self.get_test_input('TestSparseCFF2VF.otf')
```

Comment:
this `get_test_input` is method is no longer used in this class so you may well remove it